Table.gs

```diff
@@ -414,4 +414,19 @@ GridArray.prototype.limit = function(x) {
   }
 };
 
+/**
+ * Method to trim a range. The range should contain a header in the first row.
+ * @param {Range} range: a range object from Google spreadsheet. First row of range must be the headers.
+ * @returns {Range}
+ */
+Table.prototype.trimRange = function trimRange(range) {
```

Comment:
Anyways why is trimRange() a method of Table? It's a completely static method. I suggest rewriting this line to simply:
function trimRange(range) {

Also idea: Perhaps renaming to "trimRangeRows"?, since we are only trimming those? (Also put that into the description, can be misleading)

Reply:
Has it been tested properly?